frontend/lib/src/helper/snackbar/show_snackbar.dart

```diff
@@ -0,0 +1,24 @@
+import 'package:flutter/material.dart';
+import 'package:get/get.dart';
+
+/*
+  Reusable snackbar dialog
+
+  @param title: String value to display the title of the alert dialog
```

Comment:
```suggestion
  @param title: String value to display the title of the alert dialog??
```

it this alert or snackbar?